### PR TITLE
Create ku_IQ.tsx

### PR DESCRIPTION
**Pull request by ahalimkara**
Kurdish translation for Kurdistan Region of Iraq.

### CODE REVIEW

**Comment by afc163** [comment]
You should PR to feature-3.2

File: components/locale-provider/ku_IQ.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import Pagination from 'rc-pagination/lib/locale/en_GB';
```
Comment (yutingzhao1991) [inline components/locale-provider/ku_IQ.tsx]: rc-pagination@1.15.0 added locale/ku_IQ

Comment (ahalimkara) [reply components/locale-provider/ku_IQ.tsx]: May I request rc-pagination@1.15.0 in package.json? I am not sure about change logs from v1.14.0 to v1.15.0, there is no changelogs [here](https://github.com/react-component/pagination/releases). If there is no breaking changes then I will send a PR.

**Comment by ahalimkara** [comment]
@afc163 Done

File: components/locale-provider/ku_IQ.tsx (hunk 1 above)
Comment (yutingzhao1991) [reply components/locale-provider/ku_IQ.tsx]: Yes, No breaking changes and we will update rc-pagination to 1.15.0 in 3.1.4 in few day.

**Pull request closed without merging**